### PR TITLE
[incubator/azuremonitor-containers] updates related to agent 03022020 release

### PR DESCRIPTION
**Pull request by ganga1980**
#### What this PR does / why we need it:
This PR has the changes for azure monitor for containers agent 03022020 release
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: incubator/azuremonitor-containers/templates/omsagent-daemonset.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
        {{- if eq (.Values.omsagent.domain | lower) "opinsights.azure.cn" }}
        image: "dockerhub.azk8s.cn/microsoft/oms:{{ .Values.omsagent.image.tag }}"
        {{- else }}
-       image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:{{ .Values.omsagent.image.tag }}"
+       image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:{{ .Values.omsagent.image.tag }}"
```
Comment (lachie83) [inline incubator/azuremonitor-containers/templates/omsagent-daemonset.yaml]: whitespace I'd prefer to not check in :)

File: incubator/azuremonitor-containers/templates/omsagent-deployment.yaml [hunk 2]
```diff
@@ -34,7 +34,7 @@ spec:
        {{- if  eq (.Values.omsagent.domain | lower) "opinsights.azure.cn" }}
        image: "dockerhub.azk8s.cn/microsoft/oms:{{ .Values.omsagent.image.tag }}"
        {{- else }}
-       image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:{{ .Values.omsagent.image.tag }}"
+       image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:{{ .Values.omsagent.image.tag }}"
```
Comment (lachie83) [inline incubator/azuremonitor-containers/templates/omsagent-deployment.yaml]: More whitespace

File: incubator/azuremonitor-containers/templates/omsagent-daemonset.yaml (hunk 1 above)
Comment (ganga1980) [reply incubator/azuremonitor-containers/templates/omsagent-daemonset.yaml]: > whitespace I'd prefer to not check in :)
> thanks for catching this. just pushed the update which removes all the whitespaces.

File: incubator/azuremonitor-containers/templates/omsagent-deployment.yaml (hunk 2 above)
Comment (ganga1980) [reply incubator/azuremonitor-containers/templates/omsagent-deployment.yaml]: > More whitespace

Fixed as well. thanks.

**Review by lachie83 — APPROVED** [review]

**Pull request merged**